Cargo.toml

```diff
@@ -25,6 +25,7 @@ panic = "abort"
 # TODO: remove these after a new librustzcash release.
 # These are librustzcash requirements specified in its workspace Cargo.toml that we must replicate here
 incrementalmerkletree = { git = "https://github.com/zcash/incrementalmerkletree", rev = "b7bd6246122a6e9ace8edb51553fbf5228906cbb" }
-orchard = { git = "https://github.com/zcash/orchard.git", rev = "2c8241f25b943aa05203eacf9905db117c69bd29" }
+# TODO: replace with upstream orchard when these changes are merged
+orchard = { git = "https://github.com/dconnolly/orchard.git", rev = "568e24cd5f129158375d7ac7d98c89ebff4f982f" }
```

Comment:
Do we want to merge this PR without the upstream orchard changes getting merged?

If we do, let's:
* push the commit we're depending on to a `ZcashFoundation/orchard` fork
* update the dependencies and remove the `dconnolly/orchard` from the allowed sources
* open a ticket to remove the forked dependency in the next sprint (because that's when the orchard release should be cut)

Reply:
> * push the commit we're depending on to a `ZcashFoundation/orchard` fork

Done!  https://github.com/ZcashFoundation/orchard/tree/patch-1

> * update the dependencies

Done! f9d4b78b

> * remove `dconnolly/orchard` from Zebra's allowed sources

Done! f9d4b78b

> * open a ticket to remove the forked `orchard` in the next sprint (because that's when the orchard release should be cut)

Done! https://github.com/ZcashFoundation/zebra/issues/3056

Reply:
And a link in the top level Cargo.toml

f7bf21a2